main/test/ca/uwaterloo/flix/language/phase/TestWeeder.scala

```diff
@@ -915,7 +974,8 @@ class TestWeeder extends AnyFunSuite with TestUtils {
     val result = compile(input, Options.TestWithLibNix)
     expectError[WeederError.IllegalModuleName](result)
   }
-
+  // TODO: Move. Parse error "Expected UppercaseName found LowerCaseName"
+  // NOTE: Maybe there is a little too many IllegalModuleName tests?
```

Comment:
I think so. This is silly. I am not even sure why these tests are useful. I would actually consider deleting all of them.

Reply:
Or maybe keep like 3